examples/readiness-probe-pod

```diff
@@ -0,0 +1,19 @@
+# Copy of pod.yaml without file extension for test
```

Comment:
(note that I can't find any reference to those two files being used in any e2e tests anymore, so we probably don't need those lines...)